### PR TITLE
Updated getting started guide

### PR DESCRIPTION
**Pull request by StephenHodgson**
[Preview here](https://github.com/StephenHodgson/HoloToolkit-Unity/blob/HTK-GettingStartedUpdate/GettingStarted.md)

### CODE REVIEW

File: GettingStarted.md [hunk 1]
```diff
@@ -52,15 +62,17 @@ Add an `Event System` to your scene by right click on 'Managers' object in your
 
 Optionally, if you wish to enable spatial mapping in your scene, you can add the `SpatialMapping.prefab` (found under HoloToolkit/SpatialMapping/Prefabs) to your 'Managers' object.  Be aware that you must also enable `Spatial Perception` Capabilities: `Edit/Project Settings/Player -> Inspector -> Publishing Settings/Capabilities`.
 
-# 4. Building Your Project for HoloLens
-
-`HoloToolkit -> Build Window -> Build Visual Studio SLN`
-
-`Open SLN`
+# Building your project for HoloLens
```
Comment (NeerajW) [inline GettingStarted.md]: It's worth documenting the normal route for building as well.

Comment (NeerajW) [inline GettingStarted.md]: File > Build Settings etc.

File: GettingStarted.md [hunk 2]
```diff
@@ -1,40 +1,50 @@
...
+![Visual Studio Components](/External/ReadMeImages/VisualStudioComponents.PNG)
 
-# 1c. Preparing to use the HoloToolkit-Unity package
+If you haven't already installed Unity 3d on your machine, you should [download and install the latest version](https://unity3d.com/get-unity/download/archive) this project says it supports on the [main readme page](/README.md).
+
+_Note: Be sure to include the Windows Store .NET scripting backend components._
+
+![Unity Installer](/External/ReadMeImages/UnityInstaller.PNG "Unity Installer")
+
+# Download the HoloToolkit-Unity asset packages
```
Comment (NeerajW) [inline GettingStarted.md]: This seems like one long text blob. Adding numbers / bullets will help here.

Comment (StephenHodgson) [inline GettingStarted.md]: Done

File: GettingStarted.md (hunk 1 above)
Comment (StephenHodgson) [inline GettingStarted.md]: Not sure if I see a valid reason to add the normal route to building, when it's as easy as a button press. I guess we could add a link to the Unity Documentation.

**Pull request merged**